tests/test_browser.py

```diff
@@ -3738,7 +3738,7 @@ def test_pthread_volatile(self):
   # Test thread-specific data (TLS).
   @requires_threads
   def test_pthread_thread_local_storage(self):
-    self.btest(path_from_root('tests', 'pthread', 'test_pthread_thread_local_storage.cpp'), expected='0', args=['-s', 'TOTAL_MEMORY=64MB', '-O3', '-s', 'USE_PTHREADS=1', '-s', 'PTHREAD_POOL_SIZE=8'])
```

Comment:
Do you want to commit this?

Reply:
Yes, adding `-s ASSERTIONS=1` triggers the bug with the missing DataCount section that this change is meant to fix.